tuf/ngclient/_internal/trusted_metadata_set.py

```diff
@@ -271,6 +273,10 @@ def update_snapshot(self, data: bytes) -> None:
 
         Args:
             data: unverified new snapshot metadata as bytes
+            trusted: whether to consider the given data as trusted. The data is
+                trusted if it has been verified by this function. When trusted,
+                the data will be loaded even if there is a hash/length mismatch.
+                Default is False.
```

Comment:
```suggestion
            trusted: whether data has at some point been verified by
                TrustedMetadataSet as a valid snapshot. Purpose of trusted is
                to allow loading of locally stored snapshot as intermediate
                snapshot even if hashes in current timestamp meta no longer
                match data. Default is False.
```

:shrug: My version is even longer but it does try to explain the purpose of the argument. Up to you.